lib/ibanity/id_replacer.ex

```diff
@@ -12,10 +12,6 @@ defmodule Ibanity.IdReplacer do
     ...> Ibanity.IdReplacer.replace_all(url, &String.upcase/1)
     "https://api.ibanity.com/customer/synchronizations/{SYNCHRONIZATIONID}"
 
-    iex> url = "https://api.ibanity.com/customer/synchronizations"
-    ...> Ibanity.IdReplacer.replace_all(url, &String.replace("*", String.length(&1)))
-    "https://api.ibanity.com/customer/synchronizations"
-
```

Comment:
I'm not sure what this was supposed to test, but I can see that it wouldn't work because `String.replace/2` isn't a function...

Since this is in internal module, and the only way it is used is `IdReplacer.replace_all(&Recase.to_snake/1)` , I agree with removing the faulty test case. :+1: